.circleci/config.yml

```diff
@@ -145,34 +151,107 @@ jobs:
       - store_artifacts:
           path: test-results
 
-  unpackaged-sanity:
+  linux-tarball-smoke-tests:
+    <<: *defaults
+    parameters:
+      # If true, the tests will use the LTS version of node. If false, the latest version of node will be used instead.
+      lts:
+        type: boolean
+        default: true
+    steps:
+      - attach_workspace:
+          at: ~/repo
+
+      # Upgrade to whatever version of Node we need.
+      - node/install:
+          install-npm: false
+          lts: << parameters.lts >>
+
+      # Log the version of node we're using post-upgrade.
+      - run: node -v
+
+      - run:
+          name: "Install sfdx-cli"
+          command: npm install -g sfdx-cli
+
+      # Build and install our tarball.
+      - run:
+          name: "Build and install tarball"
+          command: |
+            yarn pack --filename tarball.tgz
```

Comment:
I compared `npm package` and `yarn pack`. I believe we should use `npm package`. This was the difference when i exploded the tars into directories and compared them.

```
Only in from-yarn/package: github-actions
Only in from-npm/package: NOTICE
```

Reply:
I was using `yarn pack` because it let me specify the name of the outfile. I'll switch to `npm pack` and add a `grep`, which should be fine.